sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -698,31 +721,35 @@ public void run() {
                         iterator.remove();
                         removeAllStoredErrorLogFiles(id);
                     }
-                    return;
-                }
-
-                if (userConfirmation == ALWAYS_SEND) {
-                    StorageHelper.PreferencesStorage.putBoolean(PREF_KEY_ALWAYS_SEND, true);
-                }
-
-                Iterator<Map.Entry<UUID, ErrorLogReport>> unprocessedIterator = mUnprocessedErrorReports.entrySet().iterator();
-                while (unprocessedIterator.hasNext()) {
-                    if (shouldStopProcessingPendingErrors())
-                        return;
+                } else {
 
-                    Map.Entry<UUID, ErrorLogReport> unprocessedEntry = unprocessedIterator.next();
-                    ErrorLogReport errorLogReport = unprocessedEntry.getValue();
-                    ErrorAttachment attachment = mCrashesListener.getErrorAttachment(errorLogReport.report);
-                    if (attachment == null)
-                        MobileCenterLog.debug(LOG_TAG, "CrashesListener.getErrorAttachment returned null, no additional information will be attached to log: " + errorLogReport.log.getId().toString());
-                    else
-                        errorLogReport.log.setErrorAttachment(attachment);
-                    mChannel.enqueue(errorLogReport.log, ERROR_GROUP);
+                    if (userConfirmation == ALWAYS_SEND) {
+                        StorageHelper.PreferencesStorage.putBoolean(PREF_KEY_ALWAYS_SEND, true);
+                    }
 
-                    /* Clean up an error log file and map entry. */
-                    unprocessedIterator.remove();
-                    ErrorLogHelper.removeStoredErrorLogFile(unprocessedEntry.getKey());
+                    Iterator<Map.Entry<UUID, ErrorLogReport>> unprocessedIterator = mUnprocessedErrorReports.entrySet().iterator();
+                    while (unprocessedIterator.hasNext()) {
+                        if (shouldStopProcessingPendingErrors())
+                            break;
+
+                        Map.Entry<UUID, ErrorLogReport> unprocessedEntry = unprocessedIterator.next();
+                        ErrorLogReport errorLogReport = unprocessedEntry.getValue();
+                        ErrorAttachment attachment = mCrashesListener.getErrorAttachment(errorLogReport.report);
+                        if (attachment == null)
+                            MobileCenterLog.debug(LOG_TAG, "CrashesListener.getErrorAttachment returned null, no additional information will be attached to log: " + errorLogReport.log.getId().toString());
+                        else
+                            errorLogReport.log.setErrorAttachment(attachment);
+                        mChannel.enqueue(errorLogReport.log, ERROR_GROUP);
+
+						/* Clean up an error log file and map entry. */
```

Comment:
This is very weird. My AS configuration should use spaces instead of tabs. It is not possible!!